src/composables/state-persistor/state-persistor-layer.mapper.ts

```diff
@@ -0,0 +1,44 @@
+import { Layer } from '@/stores/map.store.model'
+import useLayers from '@/composables/layers/layers.composable'
+import useThemes from '@/composables/themes/themes.composable'
+
+const STORAGE_SEPARATOR = '-'
+
+export default function useLayerMapper() {
+  const themes = useThemes()
+  const layers = useLayers()
+  
+  function layerIdsToLayersMapper(layerIdsText: string | null) {
+    const layerIds = layerIdsText ? storageToLayerIds(layerIdsText) : []
+  
+    return layerIds
+      .map(layerId => {
+        const layer = themes.findById(parseInt(layerId, 10)) as unknown as Layer
+        return layer ? layers.initLayer(layer) : void 0
+      })
+      .filter(layer => layer)
+  }
+
+  function layersToStorage(layers: Layer[] | null): any {
```

Comment:
Inconsistent names between one direction and the other

`layerIdsToLayersMapper` seems not the opposite of `layersToStorage`